graphql/resolvers.go

```diff
@@ -3231,8 +3233,18 @@ func (r *queryResolver) MainlineCommits(ctx context.Context, options MainlineCom
 		}
 		mainlineCommitVersion := MainlineCommitVersion{}
 
+		shouldNotBeRolledUp := true
```

Comment:
can we move this whole thing to be in an utility.FromBoolPtr(v.Activated) loop rather than checking individually at 3238 and 3247? that would also make it shorter to combine if statements at 3237.